packages/celer-web-app/src/ui/components/DocLineComponent/DocLineBannerComponent.tsx

```diff
@@ -39,3 +44,29 @@ export const DocLineBannerComponent: React.FC<DocLineBannerProps> = ({docLine})=
 		</div>
 	);
 };
+
+export const NoteBannerComponent: React.FC<NoteBannerProps> = ({docLine, altNotesColor}) => {
+	const {notes, variables} = docLine;
+	const styles = useStyles();
+
+	const showTriangle = false;
+	const containerColorStyle = altNotesColor ? styles.notesAlt : styles.bannerContainerColorNotes;
+	const triangleColorStyle = styles.bannerTriangleColorNotes;
+
+	// If notes are empty, return nothing
+	if (!notes) {
+		return (
+			<div></div>
```

Comment:
Does `return nulll;` works here instead of an empty div?

Reply:
`return null` seems to work fine. For some reason I thought HTML elements handled `null` values like that in a weird way, but it seems to work.